### PR TITLE
Gunicorn support

### PR DESCRIPTION
**Pull request by jwgwalton**
Checked running locally in Gunicorn process. Deployment to cloud foundry will be included in another pull request .

To Test:

Have a running instance of the database ( docker-compose up --build) 

SET:
SQLALCHEMY_DATABASE_URI = postgres://postgres:postgres@localhost:6432/postgres 

This is left at docker defaults for local development 

RUN:
gunicorn -b 0.0.0.0:$PORT application:app

### CODE REVIEW

**Review by jcox-dev — APPROVED** [review]

File: config.py [hunk 1]
```diff
@@ -10,6 +10,7 @@ class Config(object):
     SCHEMA = os.getenv("SCHEMA", "templatesvc")
     SQLALCHEMY_DATABASE_URI = os.getenv("SQLALCHEMY_DATABASE_URI",
```
Comment (jcox-dev) [inline config.py:11]: Can this be an SQLite3 URI for testing / travis?

**Review by jcox-dev — CHANGES_REQUESTED** [review]

File: config.py (hunk 1 above)
Comment (jwgwalton) [reply config.py:11]: I use a Postgres docker instance to directly test its jsonb support.

Comment (jcox-dev) [reply config.py:11]: Looks like it doesn't translate well on travis - might need an alternative to run tests.

Comment (jwgwalton) [reply config.py:11]: It's an issue with the refactoring, creating my testing client isn't picking up the right config. Trying to sort it if not I will put up the commit without the refactoring.

Comment (benjefferies) [reply config.py:11]: We could just spin up a docker postgres on travis?

**Pull request closed without merging**